src/pages/Swap.tsx

```diff
@@ -162,21 +167,21 @@ function Swap(): ReactElement {
         ? (
             formState.currentSwapPairs
               .map(({ to, type: swapType }) => {
-                if (!tokensMap[to.symbol]) {
+                if (!tokenSymbolToTokenMap[to.symbol]) {
                   console.log("unknown symbol", { to, swapType })
```

Comment:
This has been here, but was this intentional?

Reply:
If you're talking about the console.log, David left it there so best to follow up with him.

Reply:
@hammeiam, we will leave this in here for now, but it's poised to be removed should there not be a use for it.